integration/go.mod

```diff
@@ -21,6 +21,7 @@ require (
 	github.com/plus3it/gorecurcopy v0.0.1
 	github.com/rs/zerolog v1.26.1
 	github.com/stretchr/testify v1.7.1
+	go.uber.org/atomic v1.9.0
```

Comment:
Just out of interest, why use the Uber atomic package [1] instead of the built in atomic package?

I noticed it's used a lot more in flow-go, but not sure why... :-)

```
$ find . -type f | egrep "\.go" | xargs egrep "sync/atomic" | wc -l
7

$ find . -type f | egrep "\.go" | xargs egrep "go.uber.org/atomic" | wc -l
25
```

[1] https://pkg.go.dev/go.uber.org/atomic
[2] https://pkg.go.dev/sync/atomic

Reply:
This is mostly syntax sugar so one does not forget to use atomic methods on a sync/atomic variables.